static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -596,13 +591,12 @@ function WidgetBuilder({
                 )}
               >
                 <VisualizationWrapper>
-                  <DisplayTypeOptions
-                    name="displayType"
-                    options={DISPLAY_TYPES_OPTIONS}
-                    value={state.displayType}
+                  <DisplayTypeSelector
+                    displayType={state.displayType}
                     onChange={(option: {label: string; value: DisplayType}) => {
                       setState({...state, displayType: option.value});
                     }}
+                    error={state.errors?.displayType}
```

Comment:
Had to force this in the backend, but the modal handles a display type error

![Screen Shot 2022-02-23 at 6 55 00 PM](https://user-images.githubusercontent.com/22846452/155430705-cea3b11d-199a-4928-98ef-41d0d02ec01e.png)